### DISCUSSION TITLE
How are you using instascrape in your projects?

### DISCUSSION THREAD

**Discussion by chris-greening** [chris-greening/instascrape#58] [Show and tell]
Hey everyone,

I'd love to hear about the different ways you've used instascrape in you projects!

**Comment by aymalkhalid** [comment]
I am Using instascrape for scraping influencers' data. Our app called Errandly.ly(AI-powered Marketplace) will collect user Instagram data after they have a sign in. Thanks for sharing your work.

**Reply by chris-greening** [reply]
Awesome! I can't wait to see what you guys come up with, thanks so much for your response 😄 

What platforms will your app be available on?

**Reply by aymalkhalid** [reply]
On Mobile and Web, It's a start-up I am working part-time with.  Once again thanks for the scraper you made my half work easy.

**Reply by chris-greening** [reply]
Ayy sounds good, feel free to drop a link to the app here or in a separate Discussion when it's ready 😎 

And of course! Glad my scrapers could be of service 👍

**Reply by aymalkhalid** [reply]
Yeah sure Sir thank you sir for sharing your work.
